src/v2/ion/RemediationEnum.js

```diff
@@ -0,0 +1,42 @@
+const FORMS = {
+  IDENTIFY: 'identify',
+  SELECT_IDENTIFY: 'select-identify',
+  SELECT_ENROLL_PROFILE: 'select-enroll-profile',
+  ENROLL_PROFILE: 'enroll-profile',
+
+  SELECT_AUTHENTICATOR_AUTHENTICATE: 'select-authenticator-authenticate',
+  AUTHENTICATOR_VERIFICATION_DATA: 'authenticator-verification-data',
+  CHALLENGE_AUTHENTICATOR: 'challenge-authenticator',
+
+  SELECT_AUTHENTICATOR_ENROLL: 'select-authenticator-enroll',
+  SELECT_AUTHENTICATOR_ENROLL_DATA: 'select-authenticator-enroll-data',
+  AUTHENTICATOR_ENROLLMENT_DATA: 'authenticator-enrollment-data',
+  ENROLL_AUTHENTICATOR: 'enroll-authenticator',
+  SKIP: 'skip',
+
+  SUCCESS_REDIRECT: 'success-redirect',
+  REDIRECT_IDP: 'redirect-idp',
+
+  DEVICE_CHALLENGE_POLL: 'device-challenge-poll',
+  DEVICE_APPLE_SSO_EXTENSION: 'device-apple-sso-extension',
+  CANCEL_TRANSACTION: 'cancel-transaction',
+  LAUNCH_AUTHENTICATOR: 'launch-authenticator',
+};
+
+const NEED_HIDE_CANCEL_FORMS = [
```

Comment:
Can we re-name this to something else 

- `FORMS_WITHOUT_CANCEL`
- `FORMS_WITHOUT_SIGNOUT`
- `FORMS_WITHOUT_SIGNOUT_LINK`

Reply:
like `FORMS_WITHOUT_SIGNOUT`